test/lib.url.spec.ts

```diff
@@ -3,46 +3,39 @@
 
 import { format, isAbsolute, combine, concatParams } from '../src/url';
 
-
 describe('URL Handler', () => {
-
   describe('concatParams', () => {
     it('stringify and concats params to the provided URL', () => {
-      const url    = 'https://www.foo.com/bar';
-      const params =  {
-        foo: {
-          bar: {
-            baz: 'foobarbaz'
-          }
-        }
-      };
-      const actual = concatParams(url, params)
-      const expectted = 'https://www.foo.com/bar?foo%5Bbar%5D%5Bbaz%5D=foobarbaz'
+      const url = 'https://www.foo.com/bar';
+      const params = { bar: 'bar', foo: 'foo' };
+      const actual = concatParams(url, params);
+      const expectted = 'https://www.foo.com/bar?bar=bar&foo=foo';
+      const url = 'https://www.foo.com/bar';
 
       expect(actual).toBe(expectted);
     });
 
     it('when params are an empty object it returns the same url', () => {
       const url = 'https://www.foo.com/bar';
-      const params = {}
-      const actual = concatParams(url, {})
+      const params = {};
```

Comment:
We need to fix (or add) prettier hooks. My editor formatted this file, so now we have unnecessary diff 🤷‍♂